book/website/afterword/legal-disclaimer.md

```diff
@@ -0,0 +1,46 @@
+# Legal disclaimer: The Turing Way is not legal advice!
+
+## Prerequisites
+| Prerequisite | Importance | Notes |
+| -------------|----------|------|
+| None | n/a | n/a |
+
+## Summary
+The Turing Way is a lightly opinionated guide to reproducible research. It is *not* a source of legal advice and should never be treated as such. 
+
+## Good legal advice is timely, specific, and given by a legal expert 
+The Turing Way is not these things. 
+
+The Turing Way is not written to be a source of legal expertise. 
+When laws and regulations are referenced, the information provided is heavily simplified. 
+The intent is to give you an overview of the basics, so that you will know when to check whether something you want to do has potential legal ramifications. 
+
+Don’t make any important decisions based solely on information about laws and regulations in this book..
+
+### Finding legal expertise
+If you are part of an organisation such as a university, a non-profit or a company, it is very likely that you can access legal advice either in-house or from a specialist firm that already works with your institution or organisation. 
+You should check with them first if you have any concerns.
+
+If you are not part of an institution or organisation: 
+- There are law firms that specialise in data and technology who can provide advice in many jurisdictions (see below).
+- Many governments make their laws and regulations available online: be aware, however, that these can often be difficult to understand without specialist training.
+- In some jurisdictions, there are charities and non-profits who produce guidance for complying with laws. They should be clear about the limits of the advice they are able to provide. 
+
+### Jurisdictions
+Laws are different in different places. 
+You should make sure that you are aware of which juridisctions you are operating in and that you are complying with all the laws that cover all the relevant jurisdictions. 
+If you are not sure, start by contacting a legal advisor in the place that you are currently working: they will also be able to advise on what other jurisdictions you should be thinking about.
+
+## Checklist
+[ ] Do I know who I should ask if I need legal advice or expertise?
```

Comment:
Add: Do I know what legal advice I need to ask about?

Reply:
added!